docker/Dockerfile

```diff
@@ -2,11 +2,13 @@ FROM ubuntu:20.04
 
 # Make sure that the underlying container is patched to the latest versions
 RUN apt-get update && \
-    apt-get install -y gzip tar wget
+    apt-get install -y gzip tar wget && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
_Might_ be worth adding a note that any `apt-get install`s after this Docker step will require a new `apt-get update` before being able to install a package?

Reply:
As there is no any other apt-get install yet, I prefer not to have so many comments in Dockerfile, and once another apt-get install was added, contributor will find out, right? :)